.changeset/smooth-lions-brush.md

```diff
@@ -0,0 +1,6 @@
+---
+'next-urql': minor
```

Comment:
Lets maybe keep this to patch? I wouldn’t consider this a ‘minor’ bump as it’s neither a new feature, nor a significant change, nor a potentially relevant change of behaviour? It seems to me that this PR restores/establishes expected behaviour 🙏

Reply:
Gotcha, makes sense 😄